src/index.html

```diff
@@ -1,13 +1,433 @@
+  <!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>THE MET</title>
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="./images/logo-2x.png"
+    >
```

Comment:
Need to fix code style everywhere
```suggestion
    <link
      rel="icon"
      type="image/x-icon"
      href="./images/logo-2x.png"
    >
```